README.txt

```diff
@@ -30,11 +30,63 @@ Portability
     current `best practices <http://python-future.org/compatible_idioms.html>`_.
     In this way, we fully support both Python 2 and 3.
 
-Ease of use
+**Ease of use**
     The code follows the `Python Packaging User Guide
-    <http://python-packaging-user-guide.readthedocs.org/en/latest/>`_. This will
-    make installing the ``ga4gh`` reference code very easy across a range of
-    operating systems.
+    <http://python-packaging-user-guide.readthedocs.org/en/latest/>`_. 
+    Specifically, pip is used to handle python package dependencies (see below 
+    for details). This allows for easy installation of the ``ga4gh`` reference code 
+    across a range of operating systems.
+
+
+*******************************
+Installing the reference server
+*******************************
+
+Prerequisites:
+
+* Python 2.7,
+* Berkeley DB together with include and lib files (version 4.8 or higher),
+* Virtualenv (or another python sandboxing tool) is highly recommended.
+
+General installation procedure:
+
+* Install Berkeley DB (version 48 or higher) using your system's preferred 
```

Comment:
Eliminate this parenthetical ... '48' is wrong, and you already stated the required version above